src/screens/DiscoveryScreenV2/index.js

```diff
@@ -9,7 +9,6 @@ import {Header} from '../../components';
 import {Context} from '../../context';
 import DiscoveryAction from '../../context/actions/discoveryAction';
 import FollowingAction from '../../context/actions/following';
-import KeyboardWrapper from '../../navigations/KeyboardWrapper';
 import DiscoveryRepo from '../../service/discovery';
 import {
   DISCOVERY_TAB_DOMAINS,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `KeyboardWrapper` import is a good change if it's not being used in this file. This helps to keep the code clean and maintainable.
```diff
- import KeyboardWrapper from '../../navigations/KeyboardWrapper';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->